Increase Rewards Tests Coverage

narayanprusty opened the pull request:
## Description

<!-- Describe your changes here -->

Resolves #VEN-825

## Checklist

<!--
  Any non-WIP PR should have all the checkmarks set.
  If a checkmark is not applicable to your PR, mark it as done
-->

- [x] I have updated the documentation to account for the changes in the code.
- [x] If I added new functionality, I added tests covering it.
- [x] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [x] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

chechu reviewed (commented):
Would it make sense to check the specific values we would expect? Not only checking they are bigger than 0. These values shouldn't change with changes in the implementation. They are defined by business rules. This way we would check that we implemented these business rules properly

kkirka commented on tests/hardhat/Rewards.ts:
```diff
@@ -39,11 +40,10 @@ let fakePriceOracle: FakeContract<PriceOracle>;
    * Deploying required contracts along with the poolRegistry.
    */
-  before(async function () {
+  beforeEach(async function () {
```
Might be quite slow. Maybe move it into a fixture?

narayanprusty replied:
Done

narayanprusty replied:
Done

narayanprusty commented:
> Would it make sense to check the specific values we would expect? Not only checking they are bigger than 0. These values shouldn't change with changes in the implementation. They are defined by business rules. This way we would check that we implemented these business rules properly

I agree. I have calculated the rewards and updated the tests.
